### ISSUE TITLE
How does one add extensions to the php.ini?

### ISSUE THREAD

**Issue by crazyjaco** [ChrisWiegman/Primary-Vagrant#48]
Can someone point me in the right direction?

I'm trying to install a php extension and need to add it to my php.ini file, so that it is loaded and can be used.

Is this done in Puppet somehow? I have no experience with puppet, but would like to be familiar.

Thanks.

**Comment by crazyjaco** [comment]
Looks like I found what I was looking for in init.pp.

**Comment by ChrisWiegman** [comment]
Great! Sorry I didn't get to it until now. One suggestion, add the extra modules to the site definition file so you don't have to modify init.pp (assuming you're using the dev version).

**Comment by crazyjaco** [comment]
Not sure exactly whats needed to do that.
This is the extension I am using:
https://github.com/mkoppanen/php-zbarcode

I needed to git clone and compile it, so I did that within the VM.
Is it possible to do that through Puppet?

I have it working now, but it'd be much neater to have it provisioned in case I want to quickly set it up again without all those steps.

Thanks for putting this project together! I'd love to find a way to contribute back as I use it more frequently and learn the ecosystem.
